docs/diagrams/AddInterestRefSequenceDiagram.puml

```diff
@@ -0,0 +1,37 @@
+@startuml
+!include style.puml
+skinparam SequenceReferenceBackgroundColor White
+box Logic LOGIC_COLOR_T1
+participant ":AddInterest\nCommand" as AddInterestCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant ":Person" as Person MODEL_COLOR
+end box
+
+[-> AddInterestCommand : execute()
+activate AddInterestCommand
+
+activate Model
+
+AddInterestCommand -> Model: getFilteredPersonList()
+Model --> AddInterestCommand: ObservableList<Person>
+deactivate Model
+activate Person
+AddInterestCommand -> Person: get(1)
+Person --> AddInterestCommand: personToEdit
```

Comment:
Should the return arrow be at the end of the activation bar for `personToEdit` 🤔 ?
Refer to: https://nus-cs2103-ay2223s1.github.io/website/se-book-adapted/chapters/uml.html#sequence-diagrams "[Common notation error] Activation bar too long".

Reply:
Yup this is an error I made. I made several more errors which I changed